channel_print.py

```diff
@@ -0,0 +1,314 @@
+from red_star.command_dispatcher import Command
+from red_star.plugin_manager import BasePlugin
+from red_star.rs_errors import CommandSyntaxError
+from red_star.rs_utils import respond, JsonFileDict, decode_json, split_output
+from urllib.parse import urlparse
+from urllib.request import urlopen
+import mimetypes
+import re
+import json
+from discord import File, Message
+from io import BytesIO
+
+
+class EmbedDict(dict):
+    def to_dict(self):
+        return self
+
+
+def verify_embed(embed: dict):
```

Comment:
It honestly might be best to take this out of the printer and move it to rs_utils, since it's generically useful anywhere we make embeds from user input

Reply:
That sounds like a good idea and I've been meaning to do that for a while (except with a custom embed class).

Tested it, and I can safely make it return the EmbedDict to make it work for both JSON serialization purposes and sending via `respond(msg, embed=EmbedDict())` sort of arrangement.

Should I pack the class into the function or should I leave it separate?